snowplow/iglu-client-embedded/schemas/com.metabase/embed_flow/jsonschema/1-0-1

```diff
@@ -0,0 +1,205 @@
+{
```

Comment:
To see the real diff from 1-0-0: [`3bf2aa5` (#41725)](https://github.com/metabase/metabase/pull/41725/commits/3bf2aa505b12b6d90d69db36ae7bc56417e05c45#diff-9d24da11343db0597d0aab7ac1d8788ba87557b4f2049d4cbfd4c971d7114c2b)